style.css

```diff
@@ -483,3 +494,466 @@ footer .bottom_line {
   margin-top: 1rem;
   margin-bottom: 1rem;
 }
+
+@media all and (min-width: 768px) {
+  header {
+    height: 4.5rem;
+  }
+
+  .headline {
+    flex-direction: row-reverse;
+    justify-content: flex-start;
+    gap: 5.5rem;
+    padding-top: 12rem;
+    padding-bottom: 16rem;
+    background-image: url(images_4/Illustration-header.png);
+    background-position: top right;
+    background-repeat: no-repeat;
+  }
+
+  .toolbar {
+    display: none;
+  }
+
+  .open_menu {
+    width: 85%;
+    height: auto;
+    display: flex;
+    justify-content: space-between;
+    place-items: center;
+  }
+
+  ul {
+    flex-direction: column;
+    gap: 1rem;
+    margin-left: 2.25rem;
+    padding-top: 5.5rem;
+  }
+
+  footer ul {
+    margin: 0;
+    padding: 1.5rem;
+    flex-direction: row;
+  }
+
+  .open_menu ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .open_menu a {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 133%;
+  }
+
+  .intro {
+    padding: 0;
+    margin-right: 50%;
+  }
+
+  h1 {
+    height: auto;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 113%;
+    letter-spacing: 0.37px;
+    color: #f55800;
+  }
+
+  h2 {
+    margin: 1rem 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 138%;
+    color: #fff;
+  }
+
+  p {
+    height: auto;
+    padding: 1.5rem 0 1rem 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 160%;
+    color: #f4f5f7;
+  }
+
+  .connect_with {
+    width: fit-content;
+    background-image: none;
+  }
+
+  .works {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    background: #e5e5e5;
+    align-items: center;
+  }
```

Comment:
- [ ] :thinking: In my opinion, you should stick to the design as much as possible by making sure the work section looks the same as the Figma design on the desktop version shown below :point_down: 

__You  will notice that some cards mus not show the see project button, you are not required to create the JavaScript interaction yet but, kindly make it display none with css on the desktop version for now__




your design:
![Screenshot from 2023-01-20 06-14-02](https://user-images.githubusercontent.com/97834160/213616538-c3167981-a5df-482f-ab25-8d5f39b3729e.png)



Figma:
![Screenshot from 2023-01-20 06-13-16](https://user-images.githubusercontent.com/97834160/213616562-3b92ba4c-30e7-40b2-a745-b7998234fe5e.png)

Reply:
- [ ]  Hello, You have done good work implementing the requested changes, however, there is a little more work still left to be done. I will break it down below.

- [ ] According to the Figma design, the desktop version shows 3 projects in a row. Yours is showing only 2. Please make it show 3 projects on the desktop version in a row.
_**figma**_
![proj-1](https://user-images.githubusercontent.com/99656687/213652193-287b3a1c-2e21-4f8e-be92-95d23cfea393.png)
_**Your design**_
![proj-2](https://user-images.githubusercontent.com/99656687/213652434-5041d0fc-58c7-429b-8ada-04d3b495875d.png)

- [ ]  All Projects have a transparent dark overlay on top. when you hover over a project, the overlay should disappear and reveal the project with a **see project** button.
__**how a project should appear**_
![pic-1](https://user-images.githubusercontent.com/99656687/213653269-9a2e3f4c-18fc-4660-94a6-3bb0e398a46b.png)

_**How a project should appear when hovered over**_
![pic-2](https://user-images.githubusercontent.com/99656687/213653366-d2315212-6791-4559-a8dc-a9efb73a31f7.png)